src/components/common/NftDisplayer/index.tsx

```diff
@@ -50,7 +50,12 @@ const NftDisplayer = ({
         />
       )}
       {ICNSName && (
-        <ICNSDisplayer ICNSName={ICNSName} isDetailView={isDetailView} />
+        <ICNSDisplayer
+          ICNSName={ICNSName}
+          size={
+            isDetailView ? ICNSDisplayerSizes.Big : ICNSDisplayerSizes.Small
```

Comment:
with TS there's no need to declare constants as options anymore. If you send an incorrect size, it will show an error.

`size={isDetailView ? 'big' : 'small'}`